ABCD/data_convert.py

```diff
@@ -0,0 +1,73 @@
+"""ABCD tabular data conversion from TXT to TSV format.
+
+This script finds the tabular data files as downloaded from NDA in .txt format and converts it to
+.tsv format to be in compliance with BIDS specification. The TSV files are deposited in the `phenotype/`
+directory along with data dictionaries in JSON format.
+"""
+
+import argparse
+import csv
+from pathlib import Path
+
+
+def help_text():
+    parser = argparse.ArgumentParser(formatter_class=argparse.RawTextHelpFormatter, description=__doc__)
+    args = parser.parse_args()
+    return args
+
+
+def data_summary(data_with_no_dict, dict_no_data_files):
+    print(
+        f"A total of {len(data_with_no_dict)} data files in the ABCD Release 4.0 tabular data package don't have a corresponding data dictionary.")
+    print(data_with_no_dict)
+
+    print(
+        f"A total of {len(dict_no_data_files)} data dictionaries don't have a corresponding data file in the ABCD release 4.0 tabular data package.")
+    print(dict_no_data_files)
+
+
+def main():
+    # calling help prompt
+    help_text()
+
+    # hard-coding file paths based on README instructions of dir organization
+    tab_data_dir = Path('data')
+    data_dict_dir = Path('nda_data_structure_definitions')
+    outdir = Path('phenotype')
+
+    # extract nda short names into a list
+    data_files = list(tab_data_dir.glob('*.txt'))
+    data_short_names = [i.name.split('.txt')[0] for i in
```

Comment:
I think you might want ["stem"](https://docs.python.org/3/library/pathlib.html#pathlib.PurePath.stem) here?

```suggestion
    data_short_names = [i.name.stem for i in
```